docs/data/sql_connectors.yml

```diff
@@ -178,6 +178,14 @@ aws-kinesis-firehose:
   maven: flink-connector-kinesis-firehose
   sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-aws-kinesis-firehose/$version/flink-sql-connector-aws-kinesis-firehose-$version.jar
 
+dynamodb:
+    name: Amazon DynamoDB
+    category: connector
+    versions:
+        - version: 3.0.0
+          maven: flink-connector-dynamodb
+          sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-dynamodb/3.0.0-1.16/flink-sql-connector-dynamodb-3.0.0-1.16.jar
+
```

Comment:
@zentol I could not remove this since it is required by the "downloads" table. Additionally we cannot build the SQL download URL in the `sql_connector_download_table` shortcode without knowing the name of the SQL artifact. 

This renders correctly for now, I would like to get it merged to show the DynamoDB docs ready for the release today, can improve incrementally